src/clr-addons/paged-search-result-list/paged-search-result-list.spec.ts

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2018 Porsche Informatik. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { FormsModule } from '@angular/forms';
+import { ClarityModule } from '@clr/angular';
+
+import { ClrPagedSearchResultList } from './paged-search-result-list';
+
+describe('PagerComponent', () => {
+  let component: ClrPagedSearchResultList;
+  let fixture: ComponentFixture<ClrPagedSearchResultList>;
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      declarations: [ClrPagedSearchResultList],
+      imports: [ClarityModule, FormsModule],
+    }).compileComponents();
+  }));
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(ClrPagedSearchResultList);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
+  it('should create', () => {
+    expect(component).toBeTruthy();
+  });
```

Comment:
Can we have more tests here please

Reply:
this component does not offer any methods - it is more or less just 

<pager>
<custom content>
<pager>

Do you have any specific tests in mind I could implement for this?

Reply:
Maybe a test where a page switch happens and then check whether content switched as expected.
Also please rename the test from "PagerComponent" to "ClrPagedSearchResultList", otherwise its confusing in the test results